### ISSUE TITLE
how is the imu_pose and imu_transfrom got

### ISSUE THREAD

**Issue by Lachiven** [ethz-asl/segmap#157]
@rdube I am using a velodyne lidar and imu the bag like this but for sure i cant record the imu pose.Want to know how the date create where the imu_pose from.
![image](https://user-images.githubusercontent.com/50506731/78877213-1469bd80-7a83-11ea-9ec6-7f8d9504db7d.png)
![image](https://user-images.githubusercontent.com/50506731/78877280-2ea39b80-7a83-11ea-9506-1d0f3db5b322.png)

**Comment by tobby1998** [comment]
I also look into this problem. My opinion is that  the result of imu is received from TF trees. Details can be found by searching parameter "use_odom_factor" in the project (laserslamworker.cpp).

**Comment by Lachiven** [comment]
> I also look into this problem. My opinion is that the result of imu is received from TF trees. Details can be found by searching parameter "use_odom_factor" in the project (laserslamworker.cpp).

@tobby1998  yes, but in reality world how can we get that information. Have you apply this program with your own scene? we can dicuss it on QQ my QQ number is 1353632881
